samples/word/01-basics/basic-api-call.yaml

```diff
@@ -1,7 +1,6 @@
-order: 1
```

Comment:
You probably want the order (unless you have some other means by which you want to see these sorted)